config/jobs/prom-rule-reloader/prom-rule-reloader-postsubmits.yaml

```diff
@@ -0,0 +1,34 @@
+job_template: &job_template
+  max_concurrency: 10
+  path_alias: github.com/pusher/prom-rule-reloader
+  agent: kubernetes
+  always_run: true
+  skip_report: false
+  decorate: true
+  branches:
+    - master
+    # Abuse Prow to make it run on tag pushes like v1.2.3 and v1.2.3-rc1
+    - ^v?\d+\.\d+\.\d+(-rc\d+)?$
+
+container_template: &container_template
```

Comment:
No requests/limits?

Reply:
Copied from `faros`, so if we want them, we probably should add them there, too?

Reply:
This commit sugggests otherwise https://github.com/pusher/testing/pull/11/commits/1216758edb59153f2a74ced4e41f7211967c705d 😆 Any idea why that was removed? Was it broken?

Reply:
Ah, I misunderstood the template dependencies. I'll fix that.